packages/discord.js/src/structures/Guild.js

```diff
@@ -1138,6 +1138,22 @@ class Guild extends AnonymousGuild {
     return this;
   }
 
+  /**
+   * Sets the guild's MFA level
+   * @param {GuildMFALevel} level The MFA level
+   * @param {string} reason Reason for changing the guild's MFA level
+   * @returns {Promise<Guild>}
+   */
+  async setMFALevel(level, reason) {
```

Comment:
Since we're going forward with #8026, we should consolidate this into a single parameter.